src/components/explore/AggregationsOverTimeGraph.svelte

```diff
@@ -103,34 +105,44 @@
     }
 
     menuPos = { x: e.clientX, y: e.clientY };
-    clickedHov =
-      hovered.datum.build_id === '*'
-        ? hovered.datum.version
-        : hovered.datum.build_id;
-    clickedRef = getDefaultReferencePoint();
-    clickedRef =
-      aggregationLevel === 'build_id'
-        ? clickedRef.build_id
-        : clickedRef.version;
+    let referencePoint = getDefaultReferencePoint();
+
+    clickedHov = hovered.datum.build_id;
+    clickedRef = referencePoint.build_id;
```

Comment:
I noticed we used to assign `hovered.datum.build_id` to `hovered.datum.version` but removed them in this change, what did we use `hovered.datum.version` for?

Reply:
A little above this we are (line 103) we are returning if the `aggregationLevel` is not `build_id`, so this extra code would never have been called anyway. I decided to clean it up and make it simpler.